pkg/apis/core/types.go

```diff
@@ -531,8 +531,7 @@ type PersistentVolumeClaimSpec struct {
 	// * An existing PVC (PersistentVolumeClaim)
 	// If the provisioner or an external controller can support the specified data source,
 	// it will create a new volume based on the contents of the specified data source.
-	// When the AnyVolumeDataSource feature gate is enabled, dataSource contents will be copied to dataSourceRef,
-	// and dataSourceRef contents will be copied to dataSource when dataSourceRef.namespace is not specified.
+	// DataSourceRef contents will be copied to dataSource when dataSourceRef.namespace is not specified.
```

Comment:
Why is "dataSource contents will be copied to dataSourceRef," removed?

Reply:
Oh you are right, it should not be removed, let me update it.